src/styles/utils/variable.scss

```diff
@@ -0,0 +1,10 @@
+$main-color-font: #060b35;
+$hover-color-font:#00acdc;
+$main-font-size: 12px;
+$small-font-size: 10px;
+
+:root {
```

Comment:
Move the :root selector to a separate file, e.g., 'base.scss', as it's not a variable.